base/shared/src/main/scala/scalaz/data/Forall.scala

```diff
@@ -48,26 +48,13 @@ trait ForallModule {
   object Unapply {
     implicit def unapply[G[_]]: Unapply[∀[G]] { type F[A] = G[A] } =
```

Comment:
Does this subsume all the deleted cases?

Reply:
Yes. We have [examples](https://github.com/scalaz/scalaz/blob/766c09ffc0f6ef243da94868fcd61a5c892d7a19/example/shared/src/main/scala/scalaz/example/ForallUsage.scala) that previously needed those other.